mex/backend/extracted/helpers.py

```diff
@@ -0,0 +1,33 @@
+from mex.backend.extracted.models import ExtractedItemSearch
+from mex.backend.graph.connector import GraphConnector
+
+
+def search_extracted_items_in_graph(
```

Comment:
shouldnt search result in multiple items especially when `skip` and `limit` is expected.

Reply:
it does. `ExtractedItemSearch` is just a wrapper model, and it has `items` which is a list of extracted items

Reply:
ah yeah, I see it.